xwalk.gyp

```diff
@@ -372,6 +372,17 @@
             'experimental/native_file_system/virtual_root_provider_linux.cc',
           ]
         }],  # OS=="linux"
+        ['OS=="linux"' and 'tizen==0', {
```

Comment:
This is supposed to be one string (`"OS=='linux' and tizen==0"`). And I think there already exists a block with this very check somewhere else in this file.